.reek.yml

```diff
@@ -128,3 +128,4 @@ exclude_paths:
   - config
   - db/migrate
   - vendor
+  - app/controllers/api/v1/targets_controller.rb
```

Comment:
I guess this will be removed once adjusted in the 'create' branch

Reply:
This was fix in last commit